charters/DataPortal/charter.md

```diff
@@ -0,0 +1,69 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data.
```

Comment:
>It provides the user experience (UX) for users to explore cellular resolution data.

Hi, this sounds strange. Maybe a simpler way to phrase this: 

The main goal of the Data Portal is to provide life scientists interested in the Human Cell Atlas (HCA) with an easy and efficient way to explore and use data at cellular resolution.